cli/commands/create/create_test.go

```diff
@@ -0,0 +1,78 @@
+package create
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"text/template"
+
+	"github.com/sensu/sensu-go/types"
+	"github.com/stretchr/testify/mock"
+	"github.com/stretchr/testify/require"
+
+	mockclient "github.com/sensu/sensu-go/cli/client/testing"
+	cmdtesting "github.com/sensu/sensu-go/cli/commands/testing"
+)
+
+var resourceSpecTmpl = template.Must(template.New("test").Parse(`
+{"type": "Check", "spec": {{ .Check }} }
+{"type": "Asset", "spec": {{ .Asset }} }
+{"type": "Hook", "spec": {{ .Hook }} }
```

Comment:
Do you think we could also easily support uncapitalized resource type? I did instinctively used something like `check` and got the following error:

``` bash
$ sensuctl create -f check-cpu.json
error parsing resource: error parsing spec: the named type could not be found
Error: error parsing resources
```

Reply:
Yeah, that would be just a slight modification to the template. We can support both.